### ISSUE TITLE
Color conversion not done when a text is in bold or italic

### ISSUE THREAD

**Issue by BreakerZero** [pqzx/html2docx#21]
As said in the title, I can't get the color value when a text is in bold or italic, it always returns None
What I tried:

[test.py.txt](https://github.com/pqzx/html2docx/files/7226020/test.py.txt)

My original html rendering:

![html](https://user-images.githubusercontent.com/68560793/134684584-3a47e66d-6791-4700-ba9c-be7228b53ad2.PNG)

My docx rendering : 
![myexemple](https://user-images.githubusercontent.com/68560793/134684768-55f50069-e8ca-4ff4-b39a-a3b63113e385.PNG)

I have the impression that this comes from the fact that HTML2docx does not take into account the possibility that a style can be written in a <<strong>strong</strong>> or <<i>i</i>> tag.
Any solution to solve this? 
(I'd like to avoid nesting everything in a span tag every time if possible.)

**Comment by pqzx** [comment]
Yeah, restricting style to spans isn't really ideal. For now a hacky solution is to add the following
```
if 'style' in self.tags[tag]:
    style = self.parse_dict_string(self.tags[tag]['style'])
    self.add_styles_to_run(style)
```
under the final for loop in `handle_data()` https://github.com/pqzx/html2docx/blob/9337c3950bee62a4fea5b722e7ba19c163df4d9f/htmldocx/h2d.py#L550
